charts/sourcegraph/values.yaml

```
@@ -3,27 +3,41 @@
 
 ### Common Configuration ###
 sourcegraph:
-  affinity: {}
   image:
+    # -- Global docker image tag
     defaultTag: '{{ .Chart.AppVersion }}'
+    # -- Global docker image pull policy
     pullPolicy: IfNotPresent
+    # -- Global docker image registry or prefix
     repository: index.docker.io/sourcegraph
-    # When set to true, sourcegraph.image.defaultTag is used as the default defaultTag for all services, instead of service-specific default defaultTags
+    # -- When set to true, sourcegraph.image.defaultTag is used as the default defaultTag for all services, instead of service-specific default defaultTags
     useGlobalTagAsDefault: false
-  # Mount a named secret containing docker credentials
+  # -- Mount named secrets containing docker credentials
   imagePullSecrets: []
-  # Add a global label to all resources
+  # -- Add a global label to all resources
   labels: {}
-  # When true, remove all resource stanzas, allowing the scheduler to best-fit pods. Intended for local development with limited resources.
+  # -- When true, remove all resource stanzas, allowing the scheduler to best-fit pods. Intended for local development with limited resources.
   localDevMode: false
-  # Set a custom name for the app.kubernetes.io/name annotation
+  # -- Set a custom name for the app.kubernetes.io/name annotation
   nameOverride: ""
+  # -- Affinity,
```

```suggestion
  # -- Global Affinity,
```
All these `sourcegraph.` settings are global and will apply to all services.